pages/services/[ware].js

```diff
@@ -18,9 +17,10 @@ const Service = () => {
           <ItemPage
             title={ware.name}
             description={ware.description || ware.snippet}
-            // TODO(alishaevn): update the below to an actual image once
-            // https://github.com/assaydepot/scientist_api_v2/issues/184 is completed
-            img={DEFAULT_WARE_IMAGE}
+            img={{
+              src: ware.urls.promo_image || DEFAULT_WARE_IMAGE,
+              alt: `The promotional image for ${ware.name}`,
+            }}
```

Comment:
``` js
export const DEFAULT_WARE_IMAGE = {
  src: item.src,
  alt: 'Several rows of test tubes with a liquid being put into one.',
}
```

due to the way `DEFAULT_WARE_IMAGE` is defined, this code will break on instances where we don't have a valid promo image.